www/src/scss/viewer.scss

```diff
@@ -228,7 +228,7 @@ h2 {
 .logo {
   display: none;
   height: 20%;
-  left: 5%;
+  left: 72px;
```

Comment:
This cuts off the logo for 1920x1200
![image](https://user-images.githubusercontent.com/2982784/44589966-cea59780-a76e-11e8-909b-84e85706ac7f.png)

try `95px`? I think it works for most screen resolutions.

Reply:
@tsingh777 Veerji! the transform property was causing an issue here, fixed that. Now it will be 12px from the edge on every screen.